client/src/helpers/shuffleArray.js

```diff
@@ -0,0 +1,11 @@
+// From StackOverflow: https://stackoverflow.com/questions/2450954/how-to-randomize-shuffle-a-javascript-array
+// Function to randmly shuffle the contents of array
+export default function shuffleArray(array) {
```

Comment:
Why did you choose to perform the shuffle on the frontend?

Reply:
This question relates heavily to the prior point as to the structure of the endpoints that I set up. At the time of development I had made assumptions that I viewed the resources of the app as cats and sharks and thus under this model I felt that the random combination feature was not a member of either of these categories. Under the endpoints I had used I did not see any downside within the outlined requirements of the app in performing a shuffle on the frontend since the amount of data to be shuffled was so small.

Under the proposed more RESTful endpoint structure, it would make more logical sense to perform this shuffle on the backend by creating "both" as a type to be passed within the get query parameters. By shifting the shuffle function to the backend, this may for allow for more flexibility and put less demand on the browser in the future should there be requirements to shuffle larger volumes of images or from more combinations of animal types.